cmd/restic/cmd_prune.go

```diff
@@ -255,14 +270,27 @@ func prune(opts PruneOptions, gopts GlobalOptions, repo restic.Repository, usedB
 			keepBlobs.Insert(bh)
 			stats.size.used += size
 			stats.blobs.used++
-		case keepBlobs.Has(bh): // duplicate blob
-			duplicateBlobs.Insert(bh)
+			ip.usedSize += size
+			ip.duplicateBlobs++
```

Comment:
I think `ip.usedBlobs++` and `ip.duplicateBlobs++` are swapped.

Reply:
Whoops, fixed.